### PR TITLE
Fix bad input name in jbrowse-fromdir

### PR DESCRIPTION
**Pull request by hexylena**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

@bgruening noticed that it was referencing the wrong path, leading to an empty variable and reading from /data/ (yikes!). I guess it shows how often this tool is used so maybe we just delete it?

### CODE REVIEW

**Comment by hexylena** [comment]
```
cp: can't stat '/data': No such file or directory
```

well good that the test catches it. Bad that we have no way to create a dataset with an extra_files_path I guess. I tried something I saw in velvetg's wrapper but this fails with the same error. I'm not sure how to proceed, suggestions are welcome. Should we just remove the test? @abretaud any opinions?

image

File: tools/jbrowse/jbrowse-fromdir.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@ export JBROWSE_SOURCE_DIR=\$(dirname \$(command -v prepare-refseqs.pl))/../opt/j
 cp -R \${JBROWSE_SOURCE_DIR}/* $output.files_path/ &&
 
 ## Copy in data directory
-cp -R $input.dataset.extra_files_path/data/ $output.files_path/ &&
+cp -R $input.extra_files_path/data/ $output.files_path/ &&
```
Comment (bernt-matthias) [inline tools/jbrowse/jbrowse-fromdir.xml:24]: Wan you add quotes? Here and everywhere else.

**Comment by abretaud** [comment]
Yep, testing composite data seems tricky https://github.com/galaxyproject/galaxy/pull/9885
I'm not using this tool usually, do you have some usage stats on usegalaxy.*? I guess we can merge the fix anyway

File: tools/jbrowse/jbrowse-fromdir.xml [hunk 2]
```diff
@@ -13,16 +13,16 @@
...
 export JBROWSE_SOURCE_DIR=\$(dirname \$(command -v prepare-refseqs.pl))/../opt/jbrowse  &&
 
 ## Copy in jbrowse
-cp -R \${JBROWSE_SOURCE_DIR}/* $output.files_path/ &&
+cp -R \${JBROWSE_SOURCE_DIR}/* '${output.files_path}/' &&
 
 ## Copy in data directory
-cp -R $input.dataset.extra_files_path/data/ $output.files_path/ &&
-cp $output.files_path/index.html $output
+cp -R '{$input.extra_files_path}/data/' '${output.files_path}/' &&
```
Comment (bgruening) [inline tools/jbrowse/jbrowse-fromdir.xml:24]:
```suggestion
cp -R '${input.extra_files_path}/data/' '${output.files_path}/' &&
```

**Comment by hexylena** [comment]
> I'm not using this tool usually, do you have some usage stats on usegalaxy.*?

tool | count
--- | ---
jbrowse               |  4650
jbrowse_to_standalone |    86

I don't know. I used to use it to take data from apollo and make a read-only jbrowse I could share, but, we should invest energy in the apollo setup instead probably.

**Pull request closed without merging**